distribution/helm/vector/README.md

```diff
@@ -0,0 +1,59 @@
+# Vector
+
+[Vector](https://vector.dev) is used to ship Kubernetes and host logs to multiple outputs.
+
+## Prerequisites
+
+- Kubernetes 1.10+
+
+## Note
+
+By default this chart only ships a single output to container logs.
+
+## Installing the Chart
+
+To install the chart with the release name `vector`:
+
+```bash
+$ helm install --name vector stable/vector
```

Comment:
Hmmmm

```bash
[hoverbear@OBSIDIAN vector]$ helm version
version.BuildInfo{Version:"v3.1.1", GitCommit:"afe70585407b420d0097d07b21c47dc511525ac8", GitTreeState:"clean", GoVersion:"go1.14"}
[hoverbear@OBSIDIAN vector]$ helm install --name vector stable/vector
Error: unknown flag: --name
```

Reply:
It because you use helm 3.*. It has different arguments. 
I see that README written for helm 2.*

Reply:
Yes, the command for helm 3 should be `helm install --namespace vector vector path/to/helm/vector`